govcd/vm_test.go

```diff
@@ -1448,28 +1116,103 @@ func (vcd *TestVCD) Test_UpdateVmSpecSection(check *C) {
 	check.Assert(err, IsNil)
 
 	vmSpecSection := vm.VM.VmSpecSection
-	osType := "sles10_64Guest"
-	vmSpecSection.OsType = osType
 	vmSpecSection.NumCpus = addrOf(4)
 	vmSpecSection.NumCoresPerSocket = addrOf(2)
 	vmSpecSection.MemoryResourceMb = &types.MemoryResourceMb{Configured: 768}
+	if vcd.client.Client.APIVCDMaxVersionIs(">=37.1") {
+		vmSpecSection.Firmware = "efi"
+	}
 
 	updatedVm, err := vm.UpdateVmSpecSection(vmSpecSection, "updateDescription")
 	check.Assert(err, IsNil)
 	check.Assert(updatedVm, NotNil)
 
 	//verify
-	check.Assert(updatedVm.VM.VmSpecSection.OsType, Equals, osType)
+	//check.Assert(updatedVm.VM.VmSpecSection.OsType, Equals, osType)
```

Comment:
Is this comment needed? If yes, is the line needed, what is the purpose of leaving it?